demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py

```diff
@@ -0,0 +1,36 @@
+from typing import Union
+
+
+def change_incident_to_alert(data: dict) -> dict:
+    """
+    Changes internal {name: 'Related Incidents', ... }, into {name: 'Related Alerts', ... }, see the condition below.
+    """
+    if not isinstance(data, dict):
+        raise TypeError(f"expected dictionary, got {type(data)}")
+
+    def fix_recursively(datum: Union[list, dict]) -> Union[list, dict]:
+        if isinstance(datum, dict):
+            if (
+                datum.get("id") == "relatedIncidents"
```

Comment:
I saw that in some cases there is no `id` key with this value, but there is a `type` key (see [example](https://github.com/demisto/content/blob/master/Packs/CommonTypes/Layouts/layoutscontainer-Campaign.json#L55)) - is it by design to avoid these?

Reply:
Checked with @Ni-Knight, answer is positive, Implemented accordingly, please re-review :)